client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SnapshotClientDocumentationIT.java

```diff
@@ -496,7 +499,14 @@ public void testSnapshotGetSnapshots() throws IOException {
         // end::get-snapshots-execute
 
         // tag::get-snapshots-response
-        List<SnapshotInfo> snapshotsInfos = response.getSnapshots(); // <1>
+        List<SnapshotInfo> snapshotsInfos = response.getSnapshots();
+        SnapshotInfo snapshotInfo = snapshotsInfos.get(0);
+        RestStatus restStatus = snapshotInfo.status();
+        SnapshotId snapshotId = snapshotInfo.snapshotId();
+        SnapshotState snapshotState = snapshotInfo.state();
+        List<SnapshotShardFailure> snapshotShardFailures = snapshotInfo.shardFailures();
+        long startTime = snapshotInfo.startTime();
+        long endTime = snapshotInfo.endTime();
```

Comment:
would be nice here to also add some corresponding instructions in the docs page.